invenio_rdm_records/config.py

```diff
@@ -131,6 +131,12 @@ def always_valid(identifier):
 RDM_ALLOW_RESTRICTED_RECORDS = True
 """Allow users to set restricted/private records."""
 
+#
+# Record communities
+#
+RDM_RECORD_ALWAYS_IN_COMMUNITY = False
```

Comment:
Any idea if we can find a better name to this config var?

Reply:
How about `RDM_COMMUNITY_REQUIRED` or `RDM_REQUIRE_COMMUNITY`?

Reply:
maybe to be more explicit `RDM_COMMUNITY_REQUIRED_TO_PUBLISH`?